user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg

```diff
@@ -0,0 +1,65 @@
+#-------------------------------------------------#
+#### LDO Picobilical Frame PCB A1.3 definition ####
+#-------------------------------------------------#
+
+[mcu toolhead]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+restart_method: command
+##--------------------------------------------------------------------
+
+# If you want to override the wiring of the LDO Picobilical, keep in mind that this
+# board is defined using the "toolhead" name. So you should use "pin: toolhead:PIN_NAME"
+# in your own overrides.cfg files.
+
+[include config/mcu_definitions/toolhead/LDO_Picobilical_vA1.3.cfg] # Do not remove this line
+[board_pins picobilical_mcu]
+mcu: toolhead
+aliases:
+    X_STOP=MCU_XES ,
+
+    RUNOUT_SENSOR=MCU_FS ,
+
+    E_HEATER=MCU_HE0 , E_TEMPERATURE=MCU_TH0 ,
+
+    PART_FAN=MCU_PCF , E_FAN=MCU_HEF ,
+
+    PB_PCB_TEMP=MCU_CT ,
+
+    # If you want to use both of the independent neopixel ports,
+    # comment the below line and uncomment the left/right neopixel line
+    # NOTE: YOU WILL NOT BE ABLE TO USE THE STOCK KLIPPAIN
+    # MACROS WITH THE DUAL NEOPIXEL CONFIGURATION!
+    LIGHT_NEOPIXEL=MCU_NEOPIXEL_1
+    # RIGHT_NEOPIXEL=MCU_NEOPIXEL_1 , LEFT_NEOPIXEL=MCU_NEOPIXEL_2
```

Comment:
Can you rename the second one `LIGHT_NEOPIXEL_2` and keep the original `LIGHT_NEOPIXEL` as the first one?

This way this second pin can be used without touching the original Klippain pin and avoid breaking compatibility with the original macros

Reply:
whoops, I totally missed this sorry! Will fix now